tests/pipelines/pixart_sigma/test_pixart.py

```diff
@@ -396,10 +397,10 @@ def test_pixart_1024_without_resolution_binning(self):
 
         assert not np.allclose(image_slice, no_res_bin_image_slice, atol=1e-4, rtol=1e-4)
 
-    def test_pixart_512_without_resolution_binning(self):
+    def test_pixart_512_without_resolution_binning_legacy_class(self):
```

Comment:
we are not testing the new class here?

Reply:
Being tested here:
https://github.com/huggingface/diffusers/blob/3865075b00634f7b3d7daa0b2906baa3745b0bbd/tests/pipelines/pixart_sigma/test_pixart.py#L348

And here:
https://github.com/huggingface/diffusers/blob/3865075b00634f7b3d7daa0b2906baa3745b0bbd/tests/pipelines/pixart_sigma/test_pixart.py#L57

I thought that would suffice?